config/main.py

```diff
@@ -890,6 +883,17 @@ def validate_mirror_session_config(config_db, session_name, dst_port, src_port,
 @click.group(cls=AbbreviationGroup, context_settings=CONTEXT_SETTINGS)
 def config():
     """SONiC command line - 'config' command"""
+    #
+    # Load asic_type for further use
+    #
+    try:
+        version_info = sonic_device_util.get_sonic_version_info()
+        asic_type = version_info['asic_type']
```

Comment:
Should you now declare asic_type as `global` first in this function?

Reply:
you are right, it should be global.